lib/Model.php

```diff
@@ -456,8 +456,16 @@ public function assign_attribute($name, $value)
 		if ($value instanceof DateTime)
 			$value->attribute_of($this,$name);
 
+		if (array_key_exists($name,$this->attributes)) {
+			if ($this->attributes[$name] != $value) {
```

Comment:
Nie som si uplne isty. Ja chcem volat `$this->flag_dirty($name)` len ak:

1.  `array_key_exists($name,$this->attributes)` vrati false
2. nova hodnota nie je zhodna so starou

V tom pripade to nemozem takto skratit, ci? Teda dufam, ze neprehliadam nejaku zrejmu vec.